docs/tutorial/impacts.md

```diff
@@ -0,0 +1,151 @@
+
+To learn more on how we estimate the environmental impacts and what are our hypotheses go to the [methodology](../methodology/index.md) section.
+
+```python title="Structure of Impacts model"
+from ecologits.impacts.modeling import *
+
+Impacts(
+    energy=Energy(), # (1)!
+    gwp= GWP(),
+    adpe=ADPe(),
+    pe=PE(),
+    usage=Usage( # (2)!
+        energy=Energy(),
+        gwp=GWP(),
+        adpe=ADPe(),
+        pe=PE(),
+    ),
+    embodied=Embodied( # (3)!
+        gwp=GWP(),
+        adpe=ADPe(),
+        pe=PE(),
+    )
+)
+```
+
+1. Total impacts for all phases.
+2. Usage impacts for the electricity consumption impacts. Note that the energy is equal to the "total" energy impact.
+3. Embodied impacts for resource extract, manufacturing and transportation of hardware components allocated to the request. 
+
+You can extract an impact with:
+
+```python
+>>> response.impacts.usage.gwp.value  # (1)!
+0.34    # Expressed in kgCO2eq.
+```
+
+1. Assuming you have made an inference and get the response in an `response` object.
+
+Or you could get **value range** impact instead:
+
+```python
+>>> response.impacts.usage.gwp.value
+Range(min=0.16, max=0.48) # Expressed in kgCO2eq (1)
+```
+
+1. [`Range`][impacts.modeling.Range] are used to define intervals.
+
+## Criteria
+
+To evaluate the impact of human activities on the planet or on the climate we use criteria that usually focus on a specific issue such as **GHG emissions for global warming**, **water consumption and pollution** or the **depletion of natural resources**. We currently support three environmental impact criteria in addition with the direct energy consumption. 
+
+**Monitoring multiple criteria is useful to avoid pollution shifting**, which is defined as the transfer of pollution from one medium to another. It is a common pitfall to optimize only one criterion like GHG emissions (e.g. buying new hardware that is more energy efficient), that can lead to higher impacts on minerals and metals depletion for example. (see [encyclopedia.com :octicons-link-external-16:](https://www.encyclopedia.com/environment/educational-magazines/pollution-shifting))
+
+### Energy
+
+The [`Energy`][impacts.modeling.Energy] criterion refers to the **direct electricity consumption of GPUs, server and other equipments from the data center**. As defined the energy criteria is not an environmental impact, but it is used to estimate other impacts in the [usage phase](#usage). This criterion is expressed in **kilowatt-hour (kWh)**.
+
+??? note "Energy model attributes"
+    ::: impacts.modeling.Energy
+        options:
+            show_root_toc_entry: false
+            show_bases: false
+            show_docstring_description: false
+            docstring_section_style: list
+
+
+### Global Warming Potential (GWP)
```

Comment:
From here we should state that by default, the world mix is used. I will implement this with the PR following https://github.com/genai-impact/ecologits/issues/42

Reply:
Thanks, will add a dedicated section.